docs/getting_started/system.rst

```diff
@@ -437,49 +437,3 @@ Yielding the output:
 
 .. literalinclude:: xnvme_win_io_async_read_io_ring.out
    :language: bash
-
-
-Building SPDK backend on Windows
---------------------------------
-
-**SPDK** can be used as a backend for **xNVMe**, to leverage this interface
-first user need to compile the **SPDK** as a subproject with **xNVMe** its
-depend on Windows Platform Development Kit (WPDK) that help to enables
-applications based on the Storage Performance Development Kit (SPDK) to
-build and run as native Windows executables.
-
-Prerequisite’s to compile the ``SPDK``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Refer below mentioned script and link to install the dependencies packages,
-
-* MinGW cross compiler and libraries
-   this can be installed by running below script,
-
-   .. literalinclude:: ../../toolbox/pkgs/windows-2022.bat
-      :language: batch
-
-* :xref-windows-wsl:`Windows Subsystem of Linux (WSL)<>`
-
-Compilation and Installation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-* Configure, build and install using helper batch script ``build.bat``:
-
-.. literalinclude:: build_windows.rst
-   :language: bash
-   :lines: 2-
-
-Runtime Prerequisites
-~~~~~~~~~~~~~~~~~~~~~
-
-Refer to the DPDK links listed below to resolve runtime dependencies:
-
-* :xref-dpdk-guide-lock-pages:`Grant Lock pages in memory Privilege<>`
-
-* :xref-dpdk-guide-install-kmods:`Download DPDK kmods-driver<>`
-
-* :xref-dpdk-guide-install-virt2phys:`Install virt2phys<>`
-
-* :xref-dpdk-guide-install-netuio:`Install NetUIO<>`
```

Comment:
These should also be removed from the `conf.py` in `docs/autogen`

Reply:
Right, thanks.